django/db/migrations/operations/utils.py

```diff
@@ -14,6 +14,31 @@ def is_referenced_by_foreign_key(state, model_name_lower, field, field_name):
     return False
 
 
+def describe_operation(operation, backwards):
+    """
+    Returns a string, intended for `stdout.write`, that describes
+    a migration operation.
+    """
+    if hasattr(operation, "code"):
+        action = operation.code.__doc__ if backwards is not True else operation.reverse_code.__doc__
```

Comment:
simplify: `if backwards` (no need to explicitly test for boolean values I think)

Reply:
I wasn't 100% sure, so to be safe I checked it explicitly (despite my distaste). 

Will fix